chore: CloudQuery Product Updates #1 (#3408)
cloudquery/cloudquery: website/pages/blog/product-updates-1.md

---
title: "CloudQuery Product Updates #1"
tag: product
date: 2022/11/01
description: >-
  Monthly updates on CloudQuery product and roadmap.
author: yevgenypats
---

import { BlogHeader } from "../../components/BlogHeader"

<BlogHeader/>

Hey everyone! Since CloudQuery [V1 is out](./cloudquery-v1-release.md) we will be restarting our monthly product updates, and starting from #1 as we figured it is easier to use numbers than months :)

> ## Sources

Reviewer: I think it would be cool if we could highlight the codegen improvements that allow us to automatically regenerate all (AWS) resources so that we are able to keep up to date with AWS releases on launch day

---
title: "CloudQuery Product Updates #1"
tag: product

> date: 2022/11/01

Reviewer 2: Wrong date

description: >-
  Monthly updates on CloudQuery product and roadmap.
author: yevgenypats
---

import { BlogHeader } from "../../components/BlogHeader"

<BlogHeader/>

> Hey everyone! Since CloudQuery [V1 is out](./cloudquery-v1-release.md) we will be restarting our monthly products updates and starting from #1 as figured it easier to use number then months :)

Reviewer 3: ```suggestion
Hey everyone! Since CloudQuery [V1 is out](./cloudquery-v1-release.md) we will be restarting our monthly products updates and starting from #1 as I figured it is easier to use numbers than months :)
```

> This months we've been really cranking it at CQ so lot's of good stuff!

Reviewer 3: ```suggestion
This month we've been really cranking it at CQ so lots of good stuff!
```

> ## CLI Imprvoments

Reviewer 3: ```suggestion
## CLI Improvements
```

> * We've introduced glob to specify tables. Check out release [blog](https://www.cloudquery.io/blog/introducing-wildcard-matching-for-tables)

Reviewer 3: ```suggestion
* We've introduced wildcard matching to specify tables. Check out the [release blog](https://www.cloudquery.io/blog/introducing-wildcard-matching-for-tables)
```

We don't support full "glob" syntax, so we're calling it `wildcard matching`

> * We've introduced glob to specify tables. Check out release [blog](https://www.cloudquery.io/blog/introducing-wildcard-matching-for-tables)

Reviewer 2: `globbing` instead of "glob"? Though it's not really glob, is it? Maybe:

```suggestion
* We've added wildcard support to table matching. Check out release [blog](https://www.cloudquery.io/blog/introducing-wildcard-matching-for-tables)
```

table matching. or filtering?

## Destinations

We've added two new destinations to CQ!

> * [CSV](https://github.com/cloudquery/cloudquery/blob/main/plugins/destination/csv/README.md). See Release [Blog](https://www.cloudquery.io/blog/scaling-out-elt-with-cq-and-csv)

Reviewer 3: ```suggestion
* [CSV](https://github.com/cloudquery/cloudquery/blob/main/plugins/destination/csv/README.md). See [Release Blog](https://www.cloudquery.io/blog/scaling-out-elt-with-cq-and-csv)
```

> * [CSV](https://github.com/cloudquery/cloudquery/blob/main/plugins/destination/csv/README.md). See Release [Blog](https://www.cloudquery.io/blog/scaling-out-elt-with-cq-and-csv)

Reviewer 2: ```suggestion
* [CSV](https://github.com/cloudquery/cloudquery/blob/main/plugins/destination/csv/README.md). See [release blog](https://www.cloudquery.io/blog/scaling-out-elt-with-cq-and-csv)
```

> * [CSV](https://github.com/cloudquery/cloudquery/blob/main/plugins/destination/csv/README.md). See [Release Blog](https://www.cloudquery.io/blog/scaling-out-elt-with-cq-and-csv)

Reviewer 2: Is it `release blog` (as in the previous section) or `Release Blog` as in here (and below?)

> * [SQLite](https://github.com/cloudquery/cloudquery/tree/main/plugins/destination/sqlite). See Release [Blog](https://www.cloudquery.io/blog/announcing-cloudquery-sqlite-destination)

Reviewer 3: ```suggestion
* [SQLite](https://github.com/cloudquery/cloudquery/tree/main/plugins/destination/sqlite). See [Release Blog](https://www.cloudquery.io/blog/announcing-cloudquery-sqlite-destination)
```

> * [SQLite](https://github.com/cloudquery/cloudquery/tree/main/plugins/destination/sqlite). See Release [Blog](https://www.cloudquery.io/blog/announcing-cloudquery-sqlite-destination)

Reviewer 2: ```suggestion
* [SQLite](https://github.com/cloudquery/cloudquery/tree/main/plugins/destination/sqlite). See [release blog](https://www.cloudquery.io/blog/announcing-cloudquery-sqlite-destination)
```

Sporadic capitalization doesn't look nice. Title case should be for titles.

## Sources

### AWS

> We've added 16 New Service!

Reviewer 3: ```suggestion
We've added 16 new AWS services!
```

Reviewer 2: **16 new** even :)

> We've added 16 New Service!

Reviewer: A bunch of these are resources not services (`ECS Task Scale in Protection`, `ECR Repository Policy`, `ECS Image Scan Findings`,`AppRunner Additions` )

> * [EventBridge Schedule](https://github.com/cloudquery/cloudquery/issues/4693)

Reviewer 3: ```suggestion
* [EventBridge Scheduler](https://github.com/cloudquery/cloudquery/issues/4693)
```

* [SSM Resources](https://github.com/cloudquery/cloudquery/issues/4381)
* [ECS Scale](https://github.com/cloudquery/cloudquery/issues/4688)
* [MSK](https://github.com/cloudquery/cloudquery/issues/3967)
* [AWS RAM](https://github.com/cloudquery/cloudquery/issues/3961) 
* [AWS Image Scan Findings](https://github.com/cloudquery/cloudquery/issues/3607)
* [ECR Repository Policy](https://github.com/cloudquery/cloudquery/issues/3844)
* [Appstream](https://github.com/cloudquery/cloudquery/issues/3567)
* [AppRunner Additions](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/CHANGELOG.md#460-2022-11-06)
* [KMS Key Grants](https://github.com/cloudquery/cloudquery/issues/3441)
* [MWAA](https://github.com/cloudquery/cloudquery/issues/3431)
* [Service Quotas](https://github.com/cloudquery/cloudquery/issues/3489)
* [Fraud Detector Support](https://github.com/cloudquery/cloudquery/issues/3076)
* [Service Catalog](https://github.com/cloudquery/cloudquery/issues/3275)
* [App Runner](https://github.com/cloudquery/cloudquery/issues/2997)

> * [Neptune](https://github.com/cloudquery/cloudquery/issues/2923)

Reviewer 3: I haven't checked, but from memory I think `Eventbridge` is missing from this list? (It's separate from `Eventbridge Scheduler`)

> Also, tons of others bug fixes and performance tuning. See full [AWS Changelog](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/CHANGELOG.md)

Reviewer 2: ```suggestion
Also, tons of others bug fixes and performance tuning. See [full changelog](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/CHANGELOG.md)
```

### Azure

* [Functions](https://github.com/cloudquery/cloudquery/issues/3182)
* [Site Auth Settings](https://github.com/cloudquery/cloudquery/issues/3269)

> Other bugfixes as well, see full [changelog](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/azure/CHANGELOG.md)

Reviewer 2: ```suggestion
Other bug fixes as well, see [full changelog](https://github.com/cloudquery/cloudquery/blob/main/plugins/source/azure/CHANGELOG.md)
```

### K8S

> We've added 22 new resources to give a complete coverage to K8s API. Checkout all new tables [here](https://github.com/cloudquery/cloudquery/pull/4495/files#diff-ad7c419ed1d4020a168d179d12011ec7f20e5902de49d9c10d6b370f864a0d5cR1)

Reviewer 3: ```suggestion
We've added 22 new resources to give a complete coverage to K8s API. Check out all new tables [here](https://github.com/cloudquery/cloudquery/pull/4495/files#diff-ad7c419ed1d4020a168d179d12011ec7f20e5902de49d9c10d6b370f864a0d5cR1)
```

> We've added 22 new resources to give a complete coverage to K8s API. Checkout all new tables [here](https://github.com/cloudquery/cloudquery/pull/4495/files#diff-ad7c419ed1d4020a168d179d12011ec7f20e5902de49d9c10d6b370f864a0d5cR1)

Reviewer 2: ```suggestion
We've added **22 new resources** to give a complete coverage to K8s API. Check out all new tables [here](https://github.com/cloudquery/cloudquery/pull/4495/files#diff-ad7c419ed1d4020a168d179d12011ec7f20e5902de49d9c10d6b370f864a0d5cR1)
```

> ## Bugfixes

Reviewer 2: ```suggestion
## Bug Fixes
```
